casper/src/main/scala/coop/rchain/casper/util/ConstructDeploy.scala

```diff
@@ -28,15 +28,17 @@ object ConstructDeploy {
       phloLimit: Long = 90000,
       phloPrice: Long = 1L,
       sec: PrivateKey = defaultSec,
-      vabn: Long = 0
+      vabn: Long = 0,
+      shardId: String
```

Comment:
I think we can safely add empty string as default for shardId to reduce amount of changes in this PR. It will go along with other parameters which have default values.

Reply:
[Resolved](https://github.com/rchain/rchain/pull/3631/commits/ea6d84a5249a8431d0675caa816940daac7abffc).